test/correctness/saturating_casts.cpp

```diff
@@ -290,6 +290,13 @@ void test_one_source() {
 }
 
 int main(int argc, char **argv) {
+
+#ifdef __i386__
```

Comment:
I think this needs something special for MSVC, no?
```suggestion
#if defined(__i386__) || defined(_M_IX86)
```